host_applications/linux/apps/raspicam/raspiraw.c

```diff
@@ -46,6 +46,9 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <sys/ioctl.h>
 
+//Camera to use
+#define CAMERA_NUM 1
```

Comment:
Seeing as I had to check the default value, a comment along the lines of "Set to 1 on all Pi models except CM. On CM, 0=CAM0 connector, 1=CAM1 connector" would be useful.